src/UI/Tooltip.js

```diff
@@ -95,6 +96,25 @@ class Tooltip {
         }
         this.checkPosition()
     })
+    setActiveButton (e) {
+        this.activeButton = e.target
+    }
+    unsetActiveButton (e) {
+        this.activeButton = null
+    }
+    clickableButtons = new Map()
+    registerClickableButton (btn, handler) {
+        this.clickableButtons.set(btn, handler)
+        // Needed because clicks within the shadow dom don't provide this info to the outside
+        btn.addEventListener('mouseenter', (e) => this.setActiveButton(e))
+        btn.addEventListener('mouseleave', (e) => this.unsetActiveButton(e))
+    }
+    dispatchClick () {
```

Comment:
This is called by the global event handler. This function then matches the active button with its callback and fires it. Note that on mobile we're not using these web-based tooltips.